### PR TITLE
pacman-q: add alias page; pacman-query: update page

### PR DESCRIPTION
**Pull request by You-Gao**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/pacman-query.md [hunk 1]
```diff
@@ -6,32 +6,32 @@
 
 - List installed packages and versions:
```
Comment (Managor) [inline pages/linux/pacman-query.md:7]:
```suggestion
- [Q]uery the local package database and list installed packages and versions:
```
Take a look at the previous page refreshes and try to pass the meaning of the options to the mnemonics
https://github.com/tldr-pages/tldr/pull/14658/files

File: pages/linux/pacman-query.md [hunk 2]
```diff
@@ -6,32 +6,32 @@
...
 
-`pacman --query`
+`pacman -Q`
 
 - List only packages and versions that were explicitly installed:
```
Comment (Managor) [inline pages/linux/pacman-query.md:11]:
```suggestion
- List only packages and versions that were [e]xplicitly installed:
```

File: pages/linux/pacman-query.md [hunk 3]
```diff
@@ -6,32 +6,32 @@
...
 
-`pacman --query --explicit`
+`pacman -Qe`
 
 - Find which package owns a file:
```
Comment (Managor) [inline pages/linux/pacman-query.md:15]:
```suggestion
- Find which package [o]wns a file:
```

**Review by Managor — CHANGES_REQUESTED** [review]

File: pages/linux/pacman-query.md (hunk 1 above)
Comment (Managor) [reply pages/linux/pacman-query.md:7]: I don't see these issues being resolved.

**Comment by You-Gao** [comment]
mb github's mobile ui is pretty bad for prs.

I added some mnemonics, for Qdtq and Qm, their respective flags don't map cleanly to words. -t is --unrequired, -q is --quiet, -m --foreign.

File: pages/linux/pacman-query.md (hunk 1 above)
Comment (You-Gao) [reply pages/linux/pacman-query.md]: Should be good, also checked sync which answered my question below.

File: pages/linux/pacman-query.md [hunk 4]
```diff
@@ -4,34 +4,34 @@
...
+`pacman -Qi {{package}}`
 
-- List files owned by a package:
+- [L]ist files owned by a package:
 
-`pacman --query --list {{package}}`
+`pacman -Ql {{package}}`
 
-- List orphan packages (installed as dependencies but not required by any package):
+- List orphan packages (installed as [d]ependencies but unrequired ([t]) by any package without errors displayed ([q]):
```
Comment (Managor) [inline pages/linux/pacman-query.md:27]:
```suggestion
- List orphan packages (installed as [d]ependencies but unrequired ([t]) by any package without errors displayed:
```
Here the ([q]) should be attached to a word that's as close as possible to the longform word. I'm a bit hesitant on how I'd fix this.

File: pages/linux/pacman-query.md [hunk 5]
```diff
@@ -4,34 +4,34 @@
...
+`pacman -Qi {{package}}`
 
-- List files owned by a package:
+- Display the [l]ist of files owned by a specific package:
 
-`pacman --query --list {{package}}`
+`pacman -Ql {{package}}`
 
-- List orphan packages (installed as dependencies but not required by any package):
+- List orphan packages (installed as [d]ependencies but unrequired ([t]) by any package in quiet ([q]) mode (no errors displayed)):
```
Comment (Managor) [inline pages/linux/pacman-query.md:27]:
```suggestion
- List orphan packages (installed as [d]ependencies but unrequired ([t]) by any package and print in quiet ([q]) mode (no errors displayed)):
```
Looking good. I feel this is a necessary clarification.

**Review by Managor — APPROVED** [review]
Very good!

**Review by gutjuri — APPROVED** [review]
Thanks!

File: pages/linux/pacman-query.md [hunk 6]
```diff
@@ -4,34 +4,34 @@
...
+`pacman -Qi {{package}}`
 
-- List files owned by a package:
+- Display the [l]ist of files owned by a specific package:
 
-`pacman --query --list {{package}}`
+`pacman -Ql {{package}}`
 
-- List orphan packages (installed as dependencies but not required by any package):
+- List orphan packages (installed as [d]ependencies but unrequired ([t]) by any package and print in [q]uiet mode (no error messages displayed)):
```
Comment (Managor) [inline pages/linux/pacman-query.md:27]:
```suggestion
- List orphan packages (installed as [d]ependencies but unrequired ([t]) by any package and print in [q]uiet mode (only package name is displayed)):
```
One more thing. I went to the manual to make sure this was correct

**Pull request merged**